<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateItemForm</class>
 <widget class="QWidget" name="UpdateItemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update Item Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color:rgb(54,54,54);
color: white;
font: 12pt ;

}
QGroupBox{
font-weight:bold;
border:1px solid grey;}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit, QLabel{
background-color:rgb(54,54,54);
font:14px;
color:white;
}
QLineEdit#searchLineEdit{
border-radius:3px;
border:1px solid grey;
}
QPushButton#addBtn{
background-color:green;
border:1px solid grey;
color:white;
font:27px;
border-radius:5px;
}
QPushButton#addBtn:hover{
border:2px solid white;
}
QPushButton#addBtn:pressed{
border:3px solid orange;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string>Search :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
        <property name="placeholderText">
         <string>Enter Item Name</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add to table</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>250</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:1px solid grey;
border-radius:10px;
font: 12pt ;
}
QComboBox{
border:1px solid grey;
font: 12pt ;
}
QPushButton#saveItemBtn{
background-color:green;
border-radius:2px;
}
QPushButton#saveItemBtn:hover{
border:2px solid white;
}
QPushButton#saveItemBtn:pressed{
border:2px solid orange;
}
QPushButton#cancelBtn{
background-color:#30b7be;
border-radius:2px;
}
QPushButton#cancelBtn:hover{
border:2px solid white;
}
QPushButton#cancelBtn:pressed{
border:2px solid red;
}
QPushButton#updateItemBtn{
background-color:orange;
border-radius:2px;
}
QPushButton#updateItemBtn:hover{
border:2px solid white;
}
QPushButton#updateItemBtn:pressed{
border:2px solid green;
}</string>
     </property>
     <property name="title">
      <string>Update Item Details</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Item Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="itemNameLineEdit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Enter Item Name</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Buying Price</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="bpLineEdit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
            <property name="placeholderText">
             <string>Enter Buying Price</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Selling Price</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="spLineEdit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
            <property name="placeholderText">
             <string>Enter Selling Price</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Item Company</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="companyLineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
            <property name="placeholderText">
             <string>Enter Item's Company Brand</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cancelBtn">
            <property name="minimumSize">
             <size>
              <width>92</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="updateItemBtn">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>  Update Item  </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>130</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>40</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <column>
      <property name="text">
       <string>Product Name</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="background">
       <color>
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>70</red>
         <green>70</green>
         <blue>70</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Buying Price</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="background">
       <color>
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>70</red>
         <green>70</green>
         <blue>70</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Selling Price</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="background">
       <color>
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>70</red>
         <green>70</green>
         <blue>70</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Item's Company</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="background">
       <color>
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>70</red>
         <green>70</green>
         <blue>70</blue>
        </color>
       </brush>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
